django-stubs/contrib/auth/base_user.pyi

```diff
@@ -1,9 +1,15 @@
+import sys
 from typing import Any, Optional, Tuple, List, overload, TypeVar
 
 from django.db.models.base import Model
 
 from django.db import models
 
+if sys.version_info[1] < 8:
```

Comment:
This could be `if sys.version_info < (3, 8):`
(I find this slightly more readable)

Reply:
👍

Reply:
done